phenopacketlab-frontend/src/app/app.component.ts

```diff
@@ -15,26 +18,33 @@ import { SidebarComponent } from './component/sidebar/sidebar.component';
 })
 export class AppComponent implements OnInit {
```

Comment:
The app.component.spec.ts fails due to the AuthService provider missing I think. Let's fix it before merging into the development branch.